add contactForm

kinseeen opened the pull request:
https://github.com/NoroffFEU/musikkforandrerliv.no/issues/1137

Added contact form code and styling.

### What does this PR do?
created a contactForm.jsx component. This component contains a form that has: 

Content:
Input fields: Name, Email, Subject, Message
Submit button
Confirmation/success message

Notes:
Validate required fields and email format
Use controlled components
Include ARIA labels for accessibility
Implement placeholder handler or mock function for form submission

### Description of Task to be completed?
The task to was to be completed was: As a user, I want to submit a message through a form, so I can contact the organization conveniently.

### How should this be manually tested?
To manually test a contact form in a React/JSX application, you should focus on ensuring that the form functions as expected, including proper validation, submission, and handling of success/error states. You can achieve this by:
1. Testing form functionality.
2. Test scenarios
3. Test the UI.

kinseeen committed:
add contactForm

Added contact form code and styling.

senbet22 commented:
Hey @kinseeen I noticed font-size is declared twice on lines 53 and 144. Also, adding cursor: pointer to the send button would improve UX. 

One small thing: it's best practice to use PascalCase for component names in .jsx, so ContactForm instead of contactForm. I see it might be confusing, because not all groups follow this standard so i will get back to you when i double check with Abi. :)

Let me know when youve made the changes and i will merge!

kinseeen committed:
fix - #1137

 font-size is declared twice on lines 53 and 144 - fixed. Font weight was declared twice.
Also added cursor-pointer to send button.
Changed name of file from contactForm.jsx to ContactForm.jsx

kinseeen committed:
Fixed PascalCase for ContactForm

kinseeen committed:
Delete contactForm.jsx

senbet22 commented:
Well done! all issues have been fixed. merging in 3 2 1.
